### PR TITLE
Feature/ecom 123

### PR DESCRIPTION
**Pull request by GaryStackSports**

### CODE REVIEW

File: tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ public function testSucessfulCase()
 			)
 			->setExternalId('1000')
 			->setAmount(20000)
+            ->setSplitAmount(10000)
```
Comment (IvanSeguraStack) [inline tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php]: indentation

File: tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php [hunk 2]
```diff
@@ -398,7 +399,9 @@ public function testSucessfulCase()
 							'payment_method' => [
 								'method' => 'id',
 								'id' => $subscription->paymentMethod()->id(),
-							]
+							],
+                            'amount' => $subscription->amount(),
```
Comment (IvanSeguraStack) [inline tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php]: indentation

File: tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php (hunk 1 above)
Comment (GaryStackSports) [reply tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php]: Fixed... Tabs vs 4 spaces. IDE uses 4 spaces, couldn't see the difference in my IDE.

File: tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php (hunk 2 above)
Comment (GaryStackSports) [reply tests/legacy/functional/PaymentPlans/PaymentPlanEditSubscriptionWithPlanTest.php]: Fixed... Tabs vs 4 spaces. IDE uses 4 spaces, couldn't see the difference in my IDE.

**Pull request merged**